install/plantuml/install-plantuml.md

```diff
@@ -16,7 +16,7 @@
     - [Done](#done)
   - [Run Remotely](#run-remotely)
     - [Set remote setting](#set-remote-setting)
-    - [Done](#done-1)
+    - [Done](#done)
```

Comment:
roll back

Reply:
Handled in [0ae93f4](https://github.com/ajktown/docs/pull/65/commits/0ae93f4a216a68affc30883ea5728bbc8364bcd2)